File: bddtests/src/main/kotlin/mocking/emis/models/Session.kt
package mocking.emis.models

data class Session(
        var sessionName: String,
        var sessionId: Int,
        var locationId: Int? = null,
        var defaultDuration: Int? = null,
        var sessionType: SessionType? = null,
        var numberOfSlots: Int? = null,
        var clinicianIds: List<Int> = emptyList(),
        var startDate: String? = null,
        var endDate: String? = null
)

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Emis/Models/Session.cs
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;

namespace NHSOnline.Backend.GpSystems.Suppliers.Emis.Models
{
    [SuppressMessage("Microsoft.Naming", "CA1724", Justification = "Deliberately matching the name specified by the GPSS")]
    public class Session
    {
        public string SessionType { get; set; }
        public string SessionName { get; set; }
        public int SessionId { get; set; }
        public int LocationId { get; set; }
        public IEnumerable<int> ClinicianIds { get; set; }
    }
}
